src/platform/backends/shared/linux/backend_utils.h

```diff
@@ -29,6 +30,10 @@ class ProcessFactory;
 
 namespace backend
 {
+using namespace std::chrono_literals;
+
+constexpr auto image_resize_timeout = std::chrono::duration_cast<std::chrono::milliseconds>(5min).count();
```

Comment:
Nice :)

Reply:
Just one thing: `const` missing (it's unintuitive, but not implied by constexpr). Something I have forgotten a few times myself (we should probably have a PR just to fix that everywhere).

Reply:
Forgot I could just do it here...

```suggestion
constexpr const auto image_resize_timeout = std::chrono::duration_cast<std::chrono::milliseconds>(5min).count();
```

Reply:
Wrong, as discused in IRC